notebooks/utils/sklearn_soft_wrapper.py

```diff
@@ -10,3 +10,6 @@ def predict(self, x):
     def soft_predict(self, x):
         return self.skl_model.predict_proba(x)
 
+    @property
```

Comment:
this was working before without having this property. Why is it needed now?

Reply:
just looked at the issue 2464. and its fine. approved

Reply:
The property was added with the comapper operator fusion handling in the optimization work so as to allow easy querying down the model wrapper stack